### PR TITLE
add Get PowerShellGet Module page for Gallery to link to

### PR DESCRIPTION
**Pull request by tylerl0706**
This content comes from the PowerShellGet README. We are adding an additional page here so that the Gallery can reference the msdn docs page on the Gallery's Home page.

### CODE REVIEW

**Comment by saldana** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[gallery/psget/psget_get_psget_module.md](https://github.com/PowerShell/PowerShell-Docs/blob/tyleonha-get-psget-module/gallery/psget/psget_get_psget_module.md) | :warning:Warning | | [Details](#user-content-gallery%2Fpsget%2Fpsget_get_psget_module.md)

<a name="gallery%2Fpsget%2Fpsget_get_psget_module.md"></a>
### [gallery/psget/psget_get_psget_module.md](https://github.com/PowerShell/PowerShell-Docs/blob/tyleonha-get-psget-module/gallery/psget/psget_get_psget_module.md)
 - **[Warning]** Author manikb,tyleonha cannot be recognized in GitHub.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C5%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201706051825547770-1251%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by saldana** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[gallery/psget/psget_get_psget_module.md](https://github.com/PowerShell/PowerShell-Docs/blob/tyleonha-get-psget-module/gallery/psget/psget_get_psget_module.md) | :warning:Warning | | [Details](#user-content-gallery%2Fpsget%2Fpsget_get_psget_module.md)

<a name="gallery%2Fpsget%2Fpsget_get_psget_module.md"></a>
### [gallery/psget/psget_get_psget_module.md](https://github.com/PowerShell/PowerShell-Docs/blob/tyleonha-get-psget-module/gallery/psget/psget_get_psget_module.md)
 - **[Warning]** Author bmanikm,tylerl0706 cannot be recognized in GitHub.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C5%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201706051839503979-1251%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[gallery/psget/psget_get_psget_module.md](https://github.com/PowerShell/PowerShell-Docs/blob/tyleonha-get-psget-module/gallery/psget/psget_get_psget_module.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C5%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201706052121359298-1251%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: gallery/psget/psget_get_psget_module.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+### Get PowerShellGet module for PowerShell versions 3.0 and 4.0
+- [PackageManagement MSI](http://go.microsoft.com/fwlink/?LinkID=746217&clcid=0x409) 
+
+### Get the latest version from PowerShell Gallery
+
+- Before updating PowerShellGet, you should always install the latest Nuget provider. To do that, run the following in an elevated PowerShell session.
+```powershell
+Install-PackageProvider Nuget –Force
+Exit
```
Comment (zjalexander) [inline gallery/psget/psget_get_psget_module.md]: i don't think the "exit" command is necessary, unless I am missing something. If it is neccessary, can you add a quick sentence explaining that?

File: gallery/psget/psget_get_psget_module.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+```
+
+#### For systems with PowerShell 5.0 (or newer) you can install both PowerShellGet 
+- To do this on Windows 10, Windows Server 2016, or any system with WMF 5.0 or 5.1 installed, run the following commands from an elevated PowerShell session.
+```powershell
+Install-Module –Name PowerShellGet –Force
+Exit
+```
+
+- Use Update-Module to get the next updated versions.
```
Comment (zjalexander) [inline gallery/psget/psget_get_psget_module.md]: "the next updated versions" does not make sense to me.

File: gallery/psget/psget_get_psget_module.md (hunk 1 above)
Comment (tylerl0706) [inline gallery/psget/psget_get_psget_module.md]: It is required, otherwise we need to provide more commands to refresh the loaded version of PowerShellGet

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[gallery/psget/psget_get_psget_module.md](https://github.com/PowerShell/PowerShell-Docs/blob/tyleonha-get-psget-module/gallery/psget/psget_get_psget_module.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C6%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201706060000146744-1251%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: gallery/psget/psget_get_psget_module.md (hunk 1 above)
Comment (JKeithB) [inline gallery/psget/psget_get_psget_module.md]: I think the Exit is in the wrong place. Save-module won't put the module in the PSModulePath, the next 2 commands do - or am I missing something? 
After the copy-item commands the user must exit & restart PowerShell before USING PSGet.

Comment (bmanikm) [inline gallery/psget/psget_get_psget_module.md]: You can't copy the dlls into PackageManagement folder as existing dlls are already loaded into the process. That's why exit is required after save-module in PS3/4.

File: gallery/psget/psget_get_psget_module.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+---
+
+Get PowerShellGet Module
+========================
+
+### PowerShellGet is an in-box module in the following releases
+- [Windows 10](https://www.microsoft.com/en-us/windows/get-windows-10) or newer
+- [Windows Server 2016](https://technet.microsoft.com/en-us/windows-server-docs/get-started/windows-server-2016) or newer
+- [Windows Management Framework (WMF) 5.0](https://www.microsoft.com/en-us/download/details.aspx?id=50395) or newer
+- [PowerShell 6.0.0-Alpha](https://github.com/PowerShell/PowerShell/releases)
```
Comment (tylerl0706) [inline gallery/psget/psget_get_psget_module.md]: changed to PowerShell 6

File: gallery/psget/psget_get_psget_module.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
+---
```
Comment (tylerl0706) [inline gallery/psget/psget_get_psget_module.md]: renamed to get_psget_module.md

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C6%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201706060252108909-1251%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: gallery/psget/psget_get_psget_module.md (hunk 1 above)
Comment (tylerl0706) [inline gallery/psget/psget_get_psget_module.md]: I can confirm these are the correct steps. Verified on Windows Server 2012 R2 machine

File: gallery/psget/psget_get_psget_module.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+### Get the latest version from PowerShell Gallery
+
+- Before updating PowerShellGet, you should always install the latest Nuget provider. To do that, run the following in an elevated PowerShell session.
+```powershell
+Install-PackageProvider Nuget –Force
+Exit
+```
+
+#### For systems with PowerShell 5.0 (or newer) you can install both PowerShellGet
```
Comment (tylerl0706) [inline gallery/psget/psget_get_psget_module.md]: fixed

File: gallery/psget/psget_get_psget_module.md (hunk 2 above)
Comment (tylerl0706) [inline gallery/psget/psget_get_psget_module.md]: fixed

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[gallery/psget/get_psget_module.md](https://github.com/PowerShell/PowerShell-Docs/blob/tyleonha-get-psget-module/gallery/psget/get_psget_module.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C6%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201706062034051501-1251%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: gallery/psget/get_psget_module.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+### Get the latest version from PowerShell Gallery
+
+- Before updating PowerShellGet, you should always install the latest Nuget provider. To do that, run the following in an elevated PowerShell session.
+```powershell
+Install-PackageProvider Nuget –Force
+Exit
+```
+
+#### For systems with PowerShell 5.0 (or newer) you can install the latest PowerShellGet 
+- To do this on Windows 10, Windows Server 2016, or any system with WMF 5.0 or 5.1 installed, run the following commands from an elevated PowerShell session.
```
Comment (tylerl0706) [inline gallery/psget/get_psget_module.md]: Done

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[gallery/psget/get_psget_module.md](https://github.com/PowerShell/PowerShell-Docs/blob/tyleonha-get-psget-module/gallery/psget/get_psget_module.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C6%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201706062052192103-1251%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by bmanikm** [comment]
@tylerl0706 Please take a look at https://github.com/OneGet/oneget/issues/289#issuecomment-306649012 .

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[gallery/psget/get_psget_module.md](https://github.com/PowerShell/PowerShell-Docs/blob/tyleonha-get-psget-module/gallery/psget/get_psget_module.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C6%5C7%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201706071843086892-1251%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by tylerl0706** [comment]
@juanpablojofre this PR is ready to be merged. Thanks!

**Pull request merged**